sources/au/victoria.json

```diff
@@ -1,7 +1,7 @@
 {
     "type": "http",
-    "data": "http://s3.amazonaws.com/data.openaddresses.io/cache/au-victoria.zip",
-    "website": "http://services.land.vic.gov.au/landchannel/content/vicmapdata?productID=1",
+    "data": "https://www.alantgeo.com.au/share/au-victoria.zip",
```

Comment:
The original URL is working again now.

That upload-cache URL would be handy, I'm just not sure about authorising it to "write all public repository data" and to "write all user data"...